src/sass/components/_input-group.scss

```diff
@@ -18,6 +18,10 @@
     z-index: map-get($z-index, z-100);
   }
 
+  &__input:focus:not(:last-child) {
+    z-index: 201;
```

Comment:
Cool, I think this looks good. One quick thing. We do have a Sass map for `z-index` values that might be good to use here instead of a hard-coded value, assuming there isn't a good reason for hard-coding. So instead of `201` maybe:

```scss
&__input:focus:not(:last-child) {
  z-index: map-get($z-index, z-300);
}
```

Reply:
Oh nice! I definitely should've noticed that.